src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to a `.scss` file. Browsers cannot interpret SCSS files directly. Please change the href attribute to point to the compiled `.css` file instead.